src/components/global/GNB.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from '@emotion/styled';
+import THEME from '@src/constants/Colors';
+import { PATHS } from '@src/constants/Paths';
+import { Paths } from '@src/types/GNB.type';
+import { useRouter } from 'next/router';
+import React from 'react';
+import { MypageIcon, TodoBoxIcon, TodoMallIcon } from '../icons/GNBIcons';
+
+const GNB = () => {
+  const { replace, pathname } = useRouter();
+
+  const handleClick = (path: Paths) => {
+    replace(path);
+  };
+
+  const isActive = (path: Paths) => {
+    return pathname === path;
+  };
+
+  return (
+    <Container>
+      <Column
+        onClick={() => {
+          handleClick(PATHS.TODOBOX);
```

Comment:
어떤 게 조금 더 어색하지 않은지 봐도 좋을 것 같아요!

```suggestion
handleClick(PATH.TODOBOX)
```

Reply:
엇 이부분은 type과는 다르게 복수형이 조금 더 맞지 않나 싶습니다! 여러개의 paths들 중에서 하나를 고르는 의미를 갖는거니까 paths가 맞을 것 같아요!

Reply:
작성자의 입장에선 고를 수 있지만 독자의 입장에선 고르는 것이 아닌 `PATH`의 속성을 띄어주는 것이라고 생각해요. 이 부분도 같이 고민을 해보면 좋을 것 같아요!